es-bs-base/scss/vue/index.scss

```diff
@@ -0,0 +1,5 @@
+@forward "./variables";
```

Comment:
Is `vue` the right namespace here? Seems like it's all `bootstrap-vue`. However given what we talked about before maybe we should move our `es-vue-base` component scss in here as well?

Reply:
We could, but I'd prefer tackling that in a follow-on ticket